darwin/util/sysinfo_darwin.mm

```diff
@@ -1,3 +1,7 @@
+#if defined(__clang__)
+#pragma clang diagnostic ignored "-Wold-style-cast"
```

Comment:
I think better to fix this by changing `get<>Version` methods to return `static_cast<int>(osver.<>)` instead of suppressing this warning.